internal/tracker/seen_test.go test fails on i686

**Describe the bug**
I am trying to package go-toml for Fedora (https://bugzilla.redhat.com/show_bug.cgi?id=2031226). Go-toml's `internal/tracker/seen_test.go` fails on i686 with the following message:
```
github.com/pelletier/go-toml/v2/internal/testsuite
?   	github.com/pelletier/go-toml/v2/internal/testsuite	[no test files]
github.com/pelletier/go-toml/v2/internal/tracker
--- FAIL: TestEntrySize (0.00s)
    seen_test.go:14: 
        	Error Trace:	seen_test.go:14
        	Error:      	"48" is not less than or equal to "24"
        	Test:       	TestEntrySize
FAIL
exit status 1
FAIL	github.com/pelletier/go-toml/v2/internal/tracker	0.004s
```
This test uses `unsafe.Sizeof` to determine the size of a type. Is it possible that this size is smaller than expected on 32-bit platforms? Pointers there will consume 32 bits, rather than 64 bits.
 
**To Reproduce**
Build go-toml on Fedora i686. A full log of this is available at https://kojipkgs.fedoraproject.org//work/tasks/4983/86184983/build.log.

**Expected behavior**
A clear and concise description of what you expected to happen, if other than "should work".

**Versions**
 - go-toml: v2.0.0-beta.8
 - go: golang-1.18-1.fc37
 - operating system: Fedora

**Additional context**

--- a/internal/tracker/seen_test.go
+++ b/internal/tracker/seen_test.go
@@ -11,5 +11,6 @@ func TestEntrySize(t *testing.T) {
 	// Validate no regression on the size of entry{}. This is a critical bit for
 	// performance of unmarshaling documents. Should only be increased with care
 	// and a very good reason.
-	require.LessOrEqual(t, 48, int(unsafe.Sizeof(entry{})))
+	maxExpectedEntrySize := 48
+	require.LessOrEqual(t, int(unsafe.Sizeof(entry{})), maxExpectedEntrySize)
 }

